src/mlpack/bindings/R/mlpack/src/boost/serialization/variant.hpp

```diff
@@ -129,7 +129,7 @@ template<class Archive, BOOST_VARIANT_ENUM_PARAMS(/* typename */ class T)>
 void load(
     Archive & ar,
     boost::variant<BOOST_VARIANT_ENUM_PARAMS(T)>& v,
-    const unsigned int version
+    std::uint32_t const version
 ){
     int which;
     typedef typename boost::variant<BOOST_VARIANT_ENUM_PARAMS(T)>::types types;
```

Comment:
I think we should be able to remove this file entirely.  @Yashwants19 correct me if I'm wrong on that.

Reply:
We can remove this file safely. I am not sure how this file left behind even after deleting the complete `boost` folder. :thinking: